common/src/main/java/com/microsoft/identity/common/internal/fido/WebAuthnJsonUtil.kt

```diff
@@ -78,5 +79,15 @@ class WebAuthnJsonUtil {
             }
             return authResponseJsonObject.toString()
         }
+
+        /**
+         * Returns a base64URL encoding of the string.
+         * @return String
+         */
+        fun String.base64UrlEncoded(): String {
+            val data: ByteArray = this.toByteArray(Charsets.UTF_8)
+            val base64: String = Base64.encodeToString(data, (Base64.URL_SAFE or Base64.NO_WRAP or Base64.NO_PADDING))
+            return base64.replace("=", "")
```

Comment:
>base64.replace("=", "") [](http://example.com/codeflow?start=19&length=23)

Do you need this? Doesn't NO_PADDING takes care of it?

Reply:
Yep, thanks for the catch. Addressed here: 9c29abbc6ca0a117eb353731e292e170fe9c7db4